sdk/docs/iot/README.md

```diff
@@ -52,15 +52,15 @@ The library targets made available via CMake are the following:
 
 ### Samples
 
-[This page](../../../sdk/samples/iot/readme.md) explains samples for the Azure Embedded C SDK IoT Hub Client and the Provisioning Clients and how to use them.
+[This page](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/README.md) explains samples for the Azure Embedded C SDK IoT Hub Client and the Provisioning Clients and how to use them.

```

Comment:
Relative links to absolute links are out of scope for this PR

Reply:
But this was a broken link

Reply:
Also i think if we leave that as a relative link then it may make our script go into infinite loop

Reply:
Since it has a reference to itself